### PR TITLE
Update custom-resource.md

### PR DESCRIPTION
**Pull request by hegessne**
i guess mcaefeeder means mls-caefeeder and on master-live-server there is live caefeeder.
preview is located on content-management-server.

### CODE REVIEW

File: docs/custom-resource.md [hunk 1]
```diff
@@ -164,8 +164,8 @@ This tables shows the component types and the client secrets they use.
 | `content-server`/`rls` | `replication` |             |           | `publisher` | To the MLS  |
 | `cae`/`live`           |               | `blueprint` | `live`    | `webserver` |             |
 | `cae`/`preview`        |               | `blueprint` | `preview` | `webserver` |             |
-| `cae-feeder`/`live`    | `caefeeder`   | `blueprint` | `live`    | `webserver` |             |
-| `cae-feeder`/`preview` | `mcaefeeder`  | `blueprint` | `preview` | `webserver` |             |
+| `cae-feeder`/`preview` | `caefeeder`   | `blueprint` | `preview` | `webserver` |             |
```
Comment (stefanbethke) [inline docs/custom-resource.md:167]: Thanks for the correction. Would you mind swapping the two lines, so the entries in the table are all in alphabetical order?

File: docs/custom-resource.md [hunk 2]
```diff
@@ -592,4 +592,4 @@ The User Changes application.
 
 The Workflow Server application.
 
-The Solr collection is `studio`.
\ No newline at end of file
+The Solr collection is `studio`.
```
Comment (stefanbethke) [inline docs/custom-resource.md:595]: This is a stray change of whitespace?

**Review by stefanbethke — CHANGES_REQUESTED** [review]

**Comment by stefanbethke** [comment]
Implemented through https://github.com/T-Systems-MMS/cmcc-operator/commit/e61aa397296f037c9477d7a825993ce4ebe014d8.

**Comment by hegessne** [comment]
Thanks!

**Pull request closed without merging**